project-api/test-artifacts/log-creator-pkg-provided-code-generator/src/main/java/io/luhee/plugins/pkg/generator/LogCodeGeneratorPkgPlugin.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Here and the other places
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://wso2.com).
```